modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -272,7 +270,7 @@ public AbstractJavaCodegen() {
         snapShotVersionOptions.put("false", "Use a Release Version");
         snapShotVersion.setEnum(snapShotVersionOptions);
         cliOptions.add(snapShotVersion);
-        cliOptions.add(CliOption.newString(TEST_OUTPUT, "Set output folder for models and APIs tests").defaultValue(DEFAULT_TEST_FOLDER));
+        cliOptions.add(CliOption.newString(TEST_OUTPUT, "Set output folder for models and APIs tests"));
```

Comment:
what the reason to remove default value?

Reply:
It creates weird things when I try it because `${project.build.directory}` was not resolved. But I can restore it if you want.

Reply:
> It creates weird things when I try it because ${project.build.directory} was not resolved. But I can restore it if you want.

I think that's for java client (or other java server genretors) so please restore it.

if needed, remove this option and re-add it in the spring codegen with a better default value (e.g. `${project.basedir}/target/generated-test-sources`